orisintel/laravel-migration-snapshot
# Fix/pgsql data dumps cannot restore when fks

### Manual test steps
1. checkout locally
2. composer require locally checked out copy in other Laravel project using PostgreSQL and FKs
3. `php artisan migrate:dump --include-data`
4. verify `database/migrations/sql/data.pgdump` file exists
5. `php artisan migrate:load`
6. verify that output contains `pg_restore: …` and data successfully restored into DB
